src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -172,11 +173,60 @@ private static FrameworkInstallation GetFromV4(RegistryKey subKey, string subKey
             return ndpKey?.GetInt("Release");
         }
 
+        // https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#minimum-version
         internal static Version GetNetFxVersionFromRelease(int release)
         {
-            _ = NetFxReleaseVersionMap.TryGetValue(release, out var version);
-            _ = Version.TryParse(version, out var parsed);
-            return parsed;
+            if (release >= 528040)
```

Comment:
Wasn't there a 4.8.1 going to come out? I'm afraid if they change their mind about 4.8 being the last we'll be falsely reporting 4.8 for a bunch of events

Reply:
We have this in Relay too: https://github.com/getsentry/relay/blob/94a78b8c5965af9935e24ff8270968da07305432/relay-general/src/store/normalize/contexts.rs#L58-L63

Reply:
Added the missing one: https://github.com/getsentry/relay/pull/1142

Reply:
> Wasn't there a 4.8.1 going to come out? I'm afraid if they change their mind about 4.8 being the last we'll be falsely reporting 4.8 for a bunch of events

that was the behavior of the previous code

Reply:
Sorry, I don't follow. The old code relied on a map: https://github.com/getsentry/sentry-dotnet-platform-abstractions/blob/2e715e9ebe33c02a9ee9a9705d2dc1481ba90e59/src/Sentry.PlatformAbstractions/FrameworkInfo.cs#L19-L43

How would it report 4.8 for future version?

Reply:
sorry. i mis-spoke. it throws for an unknown version

Reply:
Agree that's suboptimal. Could be replaced by TryGetValue and return `null` if value isn't known. 
If we agree that reporting nothing is better than reporting the wrong value

Reply:
how about i change it to also report the reg value. then for an unknown reg value it wont log a version. at least then we can debug what the new reg value means

Reply:
That Sounds better but the only fear is that bogus data would result in high cardinality value being sent over to an indexed field. We could simple log with diagnostic logger a Warn level message with the reg value saying it doesn't match and report nothing. It's better than polluting Sentry with a bunch of new Runtime Version completely off the expected format and possibly adding lots of new values

Reply:
actually given there is no upper bound, this is not possible. ie given an arbitrary higher number than 528040, we can not tell if it should be 4.8 or 4.8.1

Reply:
We could also amend at the call site, that if we get `null` back from this, we use the reg value?
Maybe I'm splitting hairs here, if bogus values are coming in, the problem is at the call site, so happy with your proposal